crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,26 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_fadedgnrine
+TAGS:   no_monster_gen transparent
+MONS:   plant, gnoll; nothing
```

Comment:
```suggestion
MONS:   plant
```

This vault is rather recognizable and placing a guaranteed gnoll early will lead to weird spoilers problems.